projects/giflib/project.yaml

```diff
@@ -0,0 +1,6 @@
+homepage: "http://giflib.sourceforge.net/"
+primary_contact: "esr@thyrsus.com"
```

Comment:
Does this person have a GitHub account? Can you @mention them please to get their consent on the pull request?